src/setup_flow.py

```diff
@@ -290,6 +298,16 @@ async def handle_configuration_mode(
                 _LOG.warning("Could not remove device from configuration: %s", choice)
                 return SetupError(error_type=IntegrationSetupError.OTHER)
             config.devices.store()
+            adb_cert_path = Path(os.environ.get("UC_CONFIG_HOME", "./config")) / "certs" / f"adb_{choice}"
+            adb_cert_path_pub = Path(os.environ.get("UC_CONFIG_HOME", "./config")) / "certs" / f"adb_{choice}.pub"
+            if adb_cert_path.exists():
+                adb_cert_path.unlink()
+            if adb_cert_path_pub.exists():
+                adb_cert_path_pub.unlink()
+            appslist_path = Path(os.environ.get("UC_CONFIG_HOME", "./config")) / f"appslist_{choice}.json"
```

Comment:
Please use the already imported `get_config_root()` function to get the configuration path to avoid duplicating logic.